blueprints/30-nodes/blueprint.py

```diff
@@ -251,15 +252,23 @@ def delete_node_binding(nodespec):
 
     try:
         node_image_dir = node_coord2image_dir(node_coord)
-        for node_file in glob.glob(node_image_dir + '/*'):
-            os.remove(node_file)
+        node_name = os.path.basename(node_image_dir)
+        node_build_dir = BP.config['FILESYSTEM_IMAGES'] + '/' + node_name
+
+        files_to_clean = glob.glob(node_image_dir + '/*')
+        files_to_clean.extend(glob.glob(node_build_dir + '/*')) # sys-images/$NODE$/*
+
+        for to_remove in files_to_clean:
+            file_utils.remove_target(to_remove)
```

Comment:
NOTE: new code here. This will clean not only tft/images/$NODE$ folder, but also sys-images/$NODE$/*